mantidproject/mantid
# Implement wrapper for QDesktopServices

Adds a wrapper class for `QDesktopServices` so that we can intercept the `openUrl` call and remove the `LD_PRELOAD` environment variable when starting the browser on Linux-based systems. Firefox crashes if tcmalloc is preloaded when it starts.

**To test:**

Needs to be tested on Linux. 
- set Firefox as the default browser through the Firefox preferences menu.
- close any running instances of Firefox
- start mantidplot through `bin/launch_mantidplot.sh` and click "Help->Ask for Help" - Firefox should so the help documentation. 

Fixes #13909.

[RELEASE NOTES](https://github.com/mantidproject/mantid/commit/e6d62eeb6491a26e511898ab623d89264c5205e5)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
